app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/SearchSoftwarePage.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.config.AppConfig
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.views.html.templates.MainTemplate
+
+@this(mainTemplate: MainTemplate)(implicit appConfig: AppConfig)
+
+@(lastUpdate: String)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(title = messages("search-software.title")) {
+  <div class="govuk-breadcrumbs">
+    <ol class="govuk-breadcrumbs__list">
+      <li class="govuk-breadcrumbs__list-item">
+        <a class="govuk-breadcrumbs__link" href="@{appConfig.guidance}">Guidance</a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item">
+        <a class="govuk-breadcrumbs__link" href="">Filter</a>
+      </li>
+    </ol>
+  </div>
```

Comment:
Could you look into if there is a breadcrumbs component we can use instead of defining it ourselves?

Reply:
I think like the back links, these might also need to be outside of the main tag on the page.
As per: https://design-system.service.gov.uk/components/breadcrumbs/#how-it-works